bladex/blade.py

```diff
@@ -626,12 +565,58 @@ def _generate_tip(self, maxDeg):
         # Returns the Face generated by each edge of the first section
         self.generated_tip = generator.GeneratedFace(edge_1)
 
+    def _generate_root(self, maxDeg):
```

Comment:
Please no camel for the arguments: maxDeg -> max_deg.
Refactor all the code please

Reply:
That's fine, I will replace maxDeg by max_deg throughout the whole code.